test/image/mocks/range_slider_legend_left.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
This mock currently looks like:

![image](https://user-images.githubusercontent.com/6675409/63044532-66c6f000-be9c-11e9-9425-d41c9e5b819b.png)